### PR TITLE
fix(operatorgroup): Remove namespaces requirement for opgroup status

### PR DESCRIPTION
**Pull request by dinhxuanvu**
Signed-off-by: Vu Dinh <vdinh@redhat.com>

### CODE REVIEW

File: deploy/ocp/manifests/0.10.0/0000_50_olm_10-operatorgroup.crd.yaml [hunk 1]
```diff
@@ -93,7 +93,6 @@ spec:
                 type: string
               type: array
           required:
-          - namespaces
```
Comment (jpeeler) [inline deploy/ocp/manifests/0.10.0/0000_50_olm_10-operatorgroup.crd.yaml]: I don't think you want to modify the versioned directories as they have already been released.

File: manifests/0000_50_olm_10-operatorgroup.crd.yaml [hunk 2]
```diff
@@ -91,7 +91,6 @@ spec:
                 type: string
               type: array
           required:
-          - namespaces
```
Comment (jpeeler) [inline manifests/0000_50_olm_10-operatorgroup.crd.yaml]: This change is probably ok, but should be automatically updated next release.

Comment (dinhxuanvu) [reply manifests/0000_50_olm_10-operatorgroup.crd.yaml]: I ran `make release` given in the past when I changed something in template I was told to run `make release`...

Comment (jpeeler) [reply manifests/0000_50_olm_10-operatorgroup.crd.yaml]: Right, so just get rid of the other changes. Running make release is an easy way to update this directory, but also updates the other directories which we don't want.

**Pull request merged**